src/classes/RD2_DataMigration_TEST.cls

```diff
@@ -1855,6 +1855,60 @@ private class RD2_DataMigration_TEST {
             'Number of Planned Installments should be set to null');
     }
 
+    /**
+     * @description Verifies if the Accounts are populated after data migration is run
+     * when household contact is only populated and account is null.
+     */
+    @IsTest
+    private static void shouldPopulateAccountWhenContactIsOnlyEntered() {
+        npe03__Recurring_Donation__c rd = getLegacyRecurringDonationBuilder(getContact().Id)
+            .withId(null)
+            .withAccount(null)
+            .build();
+        insert rd;
+
+        Id jobId = runDataMigration();
+
+        npe03__Recurring_Donation__c convertedRD = getRecords()[0];
+
+        System.assertNotEquals(null, convertedRD.npe03__Organization__c,
+             'Account in recurring donations should not be null and should be populated post Datamigration is run' + convertedRD);
+        System.assertEquals(getContact().AccountId, convertedRD.npe03__Organization__c,
+            'Account in Recurring Donations should match to the contact account in the recurring donation post Datamigration is run' );
+
+   }
+
+    /**
+     * @description Verifies if the Accounts are populated after data migration is run
```

Comment:
- Method description can be simplified: `@description Verifies Account on RD is populated when it is not already specified and there is a closed Opp on the RD`
- Test name: `shouldPopulateAccountOnRDWhenContactIsSpecifiedAndRDHasClosedOpp()`